source/clear-linux/get-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -13,13 +13,33 @@ on bare metal using a bootable USB drive.
    :Start-after: bootable-usb-linux:
    :end-before: download-cl-image
 
-.. include:: ../bootable-usb/bootable-usb-linux.rst
-   :Start-after: download-cl-image:
-   :end-before: verify-checksum
+.. include:: ../../guides/maintenance/download-image.rst
+   :Start-after: types-of-cl-images:
+   :end-before: verify-image-checksum
 
-.. include:: ../bootable-usb/bootable-usb-linux.rst
-   :Start-after: verify-checksum:
-   :end-before: copy-usb-linux
+.. include ../bootable-usb/bootable-usb-linux.rst
+..   :Start-after: download-cl-image:
+..   :end-before: verify-checksum
+
+Download the latest Clear Linux installer image
+===============================================
+
+Get the latest |CL| installer image from the `image`_ directory.  
+Look for **clear-<version number>-installer.img.xz**.

```

Comment:
Use the :file: role for filenames and use the curly brackets, "{ }", for the variable parts of the name.  In this case the line should be:
Look for the :file:\`clear-{version number}-installer.img.xz\` file.